content/docs/start/experiments.md

```diff
@@ -19,31 +19,82 @@ once they're no longer needed.
 ## Running experiments
 
 In the previous page, we learned how to tune
-[ML pipelines](/doc/start/data-pipelines) and compare the changes. Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+[ML pipelines](/doc/start/data-pipelines) and compare the changes. In this
+section we'll use a [new Get Started project][dvcgs] to illustrate
+experimentation features in DVC 2.0
```

Comment:
Reference links are [standard markdown.](https://www.markdownguide.org/basic-syntax/#reference-style-links) I use them to keep the content within 80 columns and not breaking the URL midway. Although MD engines are smart enough to glue a link like 

```
[dvc add](https://dvc.org/doc/command
-reference/add)
```

some editors may not be. Also opening the browser, finding the exact link, and copy-paste to the document is usually a distraction so I put all links at the end. Sometimes I forget, but they are easier to spot than missing links in inline style.

Reply:
I think this links might break or not supported by the link checker. I would double check this.

Reply:
> this links might break or not supported by the link checker

I thought I knew how the link checker works but just now I tried to test this in https://github.com/iterative/dvc.org/pull/2534 and that check is skipped. Tried to look into the setup but wasn't able to figure it out quickly. I think @casperdcl wrote that right? Can you advise? Thanks 🙂

Reply:
I wrote the original (bash) link checker but since then it was moved to (a TS/JS action)  https://github.com/iterative/link-check so I'm not fully up-to-date on its internals yet

Reply:
It's been a while since I've been able to work on the link check action, the link parsing is basically a glorified regexer that goes over added/modified lines of each changed file, looking for different patterns depending on the extension. It's very possible I overlooked mid-link newlines in the MD regex parser and it never came up.

Reply:
```
and [this]?

[this]: https:://example.com
```

By support I suppose we mean that it'll check both `this` and `https:://example.com`, right? Or it can be some MD linter that check the first part of it.

Reply:
> both `this` and `https:://example.com`, right?

exactly

> can be some MD linter that check the first part

the current ones (`prettier`/`eslint`) didn't find this (I'd started complaining because I'd noticed a broken/missing reference in an early version of this PR)

Reply:
OK, so I take it for now it doesn't support reference links (nor URLS in 2 lines but Prettier wouldn't allow that anyway).